app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_transition_validator.py

```diff
@@ -50,57 +51,180 @@ def test_update_publish_status_from_approved_to_review_error(self):
             "Nimbus Experiment publish_status cannot transition from Approved to Review.",
         )
 
-    def test_launch_request_while_disabled_error(self):
```

Comment:
This side-by-side view is not being very nice. It looks like this changed, but it didn't -- 

This test (`test_launch_request_while_disabled_error`) is moved into the status_next validator tests, and `test_end_enrolment_request_while_disabled_error` and `test_end_experiment_request_while_disabled_error` have not changed.